email/html/template/textLink.ftl

```diff
@@ -1,8 +1,9 @@
 <#-- BODY - link in a text paragraph  -->
+<#--  `mailto:` can't be added to the target URL. for an email link, add mailto=true  -->
 
-<#macro emailTextLink href>
+<#macro emailTextLink href mailto>
```

Comment:
<img width="1541" alt="Screenshot 2024-12-04 at 2 52 29 PM" src="https://github.com/user-attachments/assets/4b45eb19-90f5-4e80-bba3-3a76494b4701">

Looks like this makes `mailto` a required prop whenever using this macro, causing the other email links to fail to compile (tried sending a verification email and got this error)

Reply:
sorry about that, updated